js/utilities.js

```diff
@@ -1780,3 +1780,5 @@
     ];
 
     module._systemColumns = ['RID', 'RCB', 'RMB', 'RCT', 'RMT'];
+
+    module._HIDDEN_TEXT = "--hidden--";
```

Comment:
This is not used anywhere.

Reply:
Same reason as mentioned above for isHidden. I will get rid of this variable though.